### PR TITLE
Trygit ir

### PR DESCRIPTION
**Pull request by nenadcvetkovic**
My comment

### CODE REVIEW

File: basicgit.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+We are going to try git!
+For the first time!
+Ivana Radojicic
+
```
Comment (MilanaZivojnov) [inline basicgit.txt]: Moze da ide

File: basicgit.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+We are going to try git!
```
Comment (alexcosic) [inline basicgit.txt]: super

File: basicgit.txt (hunk 1 above)
Comment (ozagajac) [inline basicgit.txt]: Good one

Comment (bojancvetkovic) [inline basicgit.txt]: Zasto je ovo prazno

File: basicgit.txt (hunk 2 above)
Comment (DraganaVracar) [reply basicgit.txt]: KOmentar 123

File: basicgit.txt (hunk 1 above)
Comment (DanBastaBGD) [inline basicgit.txt]: Best!

File: basicgit.txt (hunk 2 above)
Comment (Mo-Ye) [reply basicgit.txt]: trying out commentary

File: basicgit.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+For the first time!
+Ivana Radojicic
```
Comment (iradojicic) [inline basicgit.txt]: My name!!!

File: basicgit.txt (hunk 1 above)
Comment (Mo-Ye) [inline basicgit.txt]: Pobogu

**Pull request merged**